Remove mapped tags, ensure they won't occur again

PsiOmegaDelta opened the pull request:
Note: Maps have been updated as well, this is a pre-run to ensure Travis catches the offenders.

Regex executions, in order:
```
{tag = "(.*?)";[\s]	->	{
; tag = "(.*?)"\}	->	}
; tag = "(.*?)";	->	;
{tag = "(.*?)"\}	->	(empty)
```

Locating future tags:
`[\s{]tag = "(.*?)"`

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

comma commented:
:-1:
Tags are autogenerated by DM, which is only non-painful way to get wires/pipes. They WILL be created, and requiring people to regex them out is raising bar too much.
I am all for cleaning them out automatically, but if it's manual only, this is too strict

BlueNexus commented:
👎 same reasons as above

atlantiscze commented:
I would support this, but there would have to be automated way to remove the tags. Have you considered adding it to JMerge into the cleaning phase?

Techhead0 commented:
If we're putting it in mapmerge, put it in both.

BlueNexus reviewed (dismissed):
As per the consensus, I'm blocking this until autocleaning functionality is added to both mapmerge tools.

Techhead0 commented:
Alternatively, if tags are not enforced by Travis, then we can do without some form of tag-stripping tool.

PsiOmegaDelta commented:
The non-existence of Tags will be enforced.

atlantiscze commented:
The tag autocleaning functionality is implemented in JMerge. I am not planning to implement it for the old mapmerge tools. If you don't want to use JMerge for any reason you can always clean the tags manually if Travis detects them.

I would support merging this at this point, assuming the conflicts are resolved (which should be as easy as running JMerge clean operation on the newest copy of map files)

PsiOmegaDelta committed:
Remove mapped tags, ensure they won't occur again

Regex executions, in order:
{tag = "(.*?)";[\s]	->	{
; tag = "(.*?)"\}		->	}
; tag = "(.*?)";		->	;
{tag = "(.*?)"\}		->	(empty)

Locating future tags:
[\s{]tag = "(.*?)"

comma reviewed (approved):
eh its automagical now hy not
